docs/UserGuide.md

```diff
@@ -121,6 +121,11 @@ A person is uniquely identified by his/her **telegram handle**
 A person can have any number of roles (including 0)
 </div>
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**<br/>
+Member is a special type of contact in CCAConnect, attendance related command will only work for members.
+If you are adding a new member with `add` command, please add "member" (case-insensitive) as a role for the new contact.
+</div>
```

Comment:
Could make it clearer:
`Member` is a **special reserved role** in CCAConnect. Attendance related commands will only work for contacts that have the `Member` role.<br/>
When adding/editing a person with the role of `Member`, the spelling of `Member` in `r/Member` is case-insensitive. E.g. `r/mEmBER` will also set the person to be a `Member`

Reply:
Thanks, already updated.